test/cli/suggest-typed-true/suggest-typed-true.out

```diff
@@ -157,8 +157,16 @@ class Foo
 end
 No errors! Great job.
 -------------------------
-No errors! Great job.
-# typed: strong
+suggest-typed-and-type.rb:1: You could add `# typed: strict` https://srb.help/7022
+     1 |# typed: strong
+     2 |module I1
+  Autocorrect: Done
+    suggest-typed-and-type.rb:1: Replaced with `# typed: strict
+`
```

Comment:
Do you have any idea where this newline is coming from?

Reply:
I guess maybe we unconditionally suggest putting a newline, and there just didn't used to be a suggestion here. 👍 nvm, convinced myself